.github/workflows/delivery-release-dispatch.yml

```diff
@@ -0,0 +1,18 @@
+name: delivery / release-dispatch
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  release-dispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Repository Dispatch to Docs
+        uses: peter-evans/repository-dispatch@v1
+        if
```

Comment:
This looks incomplete.

Reply:
That's... a disturbing mistake I left in. Took it out!

Reply:
From my understanding, though, this should be the only step needed. There's no need to actually load the repository, all we really need to do is be triggered at the appropriate time and send a dispatch to `buildpacks/docs`